autosynth/github.py

```diff
@@ -209,3 +361,33 @@ def get_api_label(self, repository: str, synth_path: str) -> Optional[str]:
                 return label
 
         return None
+
+
+def _handle_response_json(response: requests.Response) -> Union[Dict, List]:
+    """Helper to parse json response from GitHub.
+
+    If the response error code indicates an error (400+), try to log the
+    error message if pressent.
+
+    Arguments:
+        response {requests.Response} - The HTTP response object
+
+    Returns:
+        Union[Dict, List] - Parsed json data
+    """
+    try:
+        json = response.json()
+        if response.status_code >= 400:
+            message = json.get("message")
+            if message:
+                logger.error(
+                    f"Error making request ({response.status_code}): {message}"
+                )
+            else:
+                logger.error(f"Error making request ({response.status_code})")
+            logger.debug(json)
+            response.raise_for_status()
+        return json
+    except ValueError as e:
+        logger.error(f"Error parsing response json: {e}")
+        raise e
```

Comment:
To preserve the original stack trace, pass no arguments to raise.  In other words, the line should be
```py
        raise
```

Reply:
Done.